src/content/enterprise/administration/configuration.mdx

```diff
@@ -287,11 +288,30 @@ autoscaler:
   slackWebhook:
 ```
 
+### backend
+
+Common configuration for backend services.
+
+- `pipelineInstaller.image`: Base image for the binaries needed by the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.annotations`: Annotations to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.env`: Environment variables to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
```

Comment:
Why are we putting the `installer` configuration inside `backend`? I think it's better if we have a `installer` section the same way that we have an `api` section, and we avoid unnecessary hierarchies.

Reply:
Yes, but this is the way it's implemented in the chart from a long time ago. we should refactor that, but I would document it the way it is for now, thoughts?

Reply:
It was implemented, but it was never documented. If we want to expose this to our customers,I think  it's "cheaper" to do it now rather than to deal with a migration/breaking change later.